components/channel/include/channel.h

```diff
@@ -79,154 +84,183 @@ typedef struct broadcast {
  * @identifier: identifier string of the channel
  * @ctx: pointer to context of the callback
  * @flags: additional flags for the callback to modify behaviour
- * @callback: function to call on new data  
- */ 
+ * @callback: function to call on new data
+ */
 void
-channel_init 
-(Channel *ch, const char *identifier, void *ctx, const BaseType_t flags, Channel_callback callback);
+channel_init(Channel         *ch,
+             const char      *identifier,
+             void            *ctx,
```

Comment:
Was this spacing in here before? Personally I don't like the spacing here, as it disturbs the flow imo. Is it possible to not indent it here, but at the other places?

Reply:
do you mean the alignment of the identifiers? In my opinion this is one of the best features of uncrustify.
I could set a maximum of inserted spaces here (currently unlimited) but I really like this.